### ISSUE TITLE
Documentation Recommends Use of Deprecated Brew Java8 Installation

### ISSUE THREAD

**Issue by decodyng** [minerllabs/minerl#30]
This is more a FYI about documentation than it is a bug in code, but trying to run the commands that you recommend in the docs to install Java8 on a mac, namely: 

```
brew tap caskroom/versions
brew cask install java8 
```

errors out with 

```
Error: Cask 'java8' is unavailable: No Cask with this name exists.
```

Per this stack overflow, it appears that installing java8 via brew is now best done through AdoptOpenJDK: https://stackoverflow.com/questions/55834845/unable-to-install-java8-with-homebrew

**Comment by MadcowD** [comment]
Oh awesome find! Would you be willing to make a PR for this in the documentation?

**Comment by decodyng** [comment]
Sure, I'd be happy to

**Comment by MadcowD** [comment]
:)))))

**Comment by decodyng** [comment]
PR here: https://github.com/minerllabs/minerl/pull/32

Apologies for it containing about as many commits as there are words in the sentence I'm changing; I'm new to RST and got a bit burned by how fiddly it is.

**Comment by MadcowD** [comment]
I just learned how to do it last weekend . Believe me I know :/

**Comment by minerllabs** [comment]
Closed by #32
